pkgs/applications/misc/tpmmanager/default.nix

```diff
@@ -8,12 +8,24 @@ stdenv.mkDerivation rec {
     owner = "Rohde-Schwarz";
     repo = "TPMManager";
     rev = "v${version}";
-    sha256 = "sha256-JKYG+I/tZ+0NDmHcIgKV6eGrjbPvPQKPo0sE/zBlLY4=";
+    sha256 = "sha256-UZYn4ssbvLpdB0DssT7MXqQZCu1KkLf/Bsb45Rvgm+E=";
```

Comment:
why did the hash change, if the rev didn't?

Reply:
The old hash was probably wrong. The build failed and the hash needed updating to fix this.
The hash was last touched when this was converted to ffGH: https://github.com/NixOS/nixpkgs/commit/3a1e015627474c8f1da6a5c83577cd3e293a7475

ping @Stunkymonkey

Reply:
I can confirm, that the new hash is correct. No idea where I got mine from.